src/cswinrt/code_writers.h

```diff
@@ -5196,8 +5187,7 @@ _defaultLazy = new Lazy<%>(() => ifc);
 _lazyInterfaces = new Dictionary<Type, object>()
 {%
 };
-%}
-%
+}
```

Comment:
Do we not also want this for .net standard?

Reply:
At first, I thought not, but I agree - it should be supported there too